apps/nextjs/scripts/increase-listeners.js

```diff
@@ -0,0 +1,7 @@
+// Increase the limit of max listeners
+require("events").EventEmitter.defaultMaxListeners = 20;
+
+// Run the original command
+require("child_process").spawn(process.argv[2], process.argv.slice(3), {
+  stdio: "inherit",
+});
```

Comment:
It would be useful to have context for why this is needed, and under what conditions we would be able to remove it